bin/ci/cucumber.rb

```diff
@@ -8,99 +8,35 @@
 this_dir = File.expand_path(File.dirname(__FILE__))
 working_directory = File.join(this_dir, "..", "..")
 
+
 # on-simulator tests of features in test/cucumber
 Dir.chdir(working_directory) do
 
   FileUtils.rm_rf("reports")
   FileUtils.mkdir_p("reports")
 
-  Luffa.unix_command("make clean",
-                     {:pass_msg => "cleaned",
-                      :fail_msg => "could not clean"})
-
-  Luffa.unix_command("make app",
-                     {:pass_msg => "built app",
-                      :fail_msg => "could not build app"})
-
   xcode = RunLoop::Xcode.new
-  xcode_version = xcode.version
-  sim_major = xcode_version.major + 2
-  sim_minor = xcode_version.minor
-
-  sim_version = RunLoop::Version.new("#{sim_major}.#{sim_minor}")
-
-  if ENV["JENKINS_HOME"]
-    devices = {
-      :iphone7 => "iPhone 7"
-    }
-  else
-    devices = {
-      :iphoneSE => "iPhone SE"
-    }
-  end
-
   RunLoop::CoreSimulator.terminate_core_simulator_processes
 
-  simulators = RunLoop::Simctl.new.simulators
+  simctl = RunLoop::Simctl.new
+  instruments = RunLoop::Instruments.new
+  options = {}
+  match = RunLoop::Device.detect_device(options, xcode, simctl, instruments)
+  env_vars = {"DEVICE_TARGET" => match.udid}
 
-  env_vars = {}
+  languagesLiterals = ["en", "ru"]
 
-  passed_sims = []
-  failed_sims = []
-  devices.each do |key, name|
+  languagesLiterals.each do |item|
+    language = "APP_LANG=#{item} APP_LOCALE=#{item}"
     args = [
       "bundle", "exec",
       "cucumber", "-p", "default",
-      "-f", "json", "-o", "reports/#{key}.json",
+      "-f", "json", "-o", "reports/#{item}.json",
       "-f", "junit", "-o", "reports/junit",
```

Comment:
Test results for Russian locale will overwrite the English ones.
I propose to do something like that:
```
"-f", "junit", "-o", "reports/junit/#{key}",
```